templates/listcollectorcontent.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'layouts/_questionnaire.html' %}
+{% from "components/question/_macro.njk" import onsQuestion %}
+{% import 'macros/helpers.html' as helpers %}
+
+{% set save_on_signout = true %}
```

Comment:
You need to set `{% set continue_button_text = _("Continue") %}`

At the moment the button text is "Save and continue" which is not what we wabt. Test should be updated to verify.